components/Team.js

```diff
@@ -122,24 +122,28 @@ export default function Team({ profiles }) {
         }
         <div style={{ willChange: 'transform' }} id='anim-profiles'>
           {profiles.map((profile, i) => (
-            <ProfileImage
-              key={i}
-              src={profile.img}
-              color={profile.color}
-              onClick={() => setSelectedProfile(profile)}
-              onMouseEnter={() => setSelectedProfile(profile)}
-              onMouseLeave={() => setSelectedProfile({})}
-            />
+            <a href={profile.social}>
+              <ProfileImage
+                key={i}
+                src={profile.img}
+                color={profile.color}
+                onClick={() => setSelectedProfile(profile)}
+                onMouseEnter={() => setSelectedProfile(profile)}
+                onMouseLeave={() => setSelectedProfile({})}
+              />
+          </a>
           ))}
           {profiles.map((profile, i) => (
-            <ProfileImage
-              key={i}
-              src={profile.img}
-              color={profile.color}
-              onClick={() => setSelectedProfile(profile)}
-              onMouseEnter={() => setSelectedProfile(profile)}
-              onMouseLeave={() => setSelectedProfile({})}
-            />
+            <a href={profile.social}>
+              <ProfileImage
+                key={i}
```

Comment:
I think there are keys that aren't unique since the children in the above map could have the same key

Reply:
fixed